dependencies.yaml

```
@@ -100,6 +102,7 @@ dependencies:
       - output_types: conda
         packages:
           - c-compiler
+          - cuda-nvcc
           - cxx-compiler
```

We can consolidate this to `cuda-compiler`, which pulls in [the other compilers]( https://github.com/conda-forge/cuda-compiler-feedstock/blob/b4313ac3b6a046f95c764d4d6aecd0ed4956f334/recipe/meta.yaml#L22-L23 ) as well

```suggestion
        packages:
          - cuda-compiler
```